fig/Shell Script For Loop Flow Chart.svg

```diff
@@ -0,0 +1,1083 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>

```

Comment:
Thanks Ashwin. Deleted redundant file in fig/Shell Script For Loop Flow Chart.svg<https://github.com/swcarpentry/shell-novice/pull/366#discussion_r64202426>:: Shell Script For Loop Flow Chart.svg. fig/Shell Script For Loop Flow Chart.svg<https://github.com/swcarpentry/shell-novice/pull/366#discussion_r64202426> now only has the referenced files: shell_script_for_loop_flow_chart.svg

[cid:image001.png@01D1B5B5.11D45090]

Cheers,

Dr Francis Gacenga
QCIF eResearch Analyst
Office of Research Development | Research and Innovation Division | University of Southern
Queensland | Toowoomba, Queensland | 4350| Australia
Phone: +61 7 4687 5640 | 0477 072 707 | Email: gacenga@usq.edu.au<mailto:gacenga@usq.edu.au> | Twitter: https://twitter.com/USQ_eResearch
Website: https://www.usq.edu.au/eresearch |QRIScloud: https://www.qriscloud.org.au/

From: Ashwin Srinath [mailto:notifications@github.com]
Sent: Monday, 23 May 2016 8:48 PM
To: swcarpentry/shell-novice <shell-novice@noreply.github.com>
Cc: Francis Gacenga <Gacenga@usq.edu.au>; Mention <mention@noreply.github.com>
Subject: Re: [swcarpentry/shell-novice] Update 04-Loop.md (#366)


In fig/Shell Script For Loop Flow Chart.svg<https://github.com/swcarpentry/shell-novice/pull/366#discussion_r64202426>:

> @@ -0,0 +1,1083 @@

> +<?xml version="1.0" encoding="UTF-8" standalone="no"?>

This file is called Shell Script For Loop Flow Chart.svg, but referenced as shell_script_for_loop_flow_chart.svg. Please change the file name

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/swcarpentry/shell-novice/pull/366/files/517d6fd3fe0e4ed43e7eb7ec3d00969ec6922006#r64202426>


_____________________________________________________________
This email (including any attached files) is confidential and is for the intended recipient(s) only. If you received this email by mistake, please, as a courtesy, tell the sender, then delete this email.

The views and opinions are the originator's and do not necessarily reflect those of the University of Southern Queensland. Although all reasonable precautions were taken to ensure that this email contained no viruses at the time it was sent we accept no liability for any losses arising from its receipt.

The University of Southern Queensland is a registered provider of education with the Australian Government.
(CRICOS Institution Code QLD 00244B / NSW 02225M, TEQSA PRV12081 )